### PR TITLE
Customizable footer layout

### PR DESCRIPTION
**Pull request by ProphetLamb**
The specification document of the DHBW Mosbach, defines that the pagemark should be located on the right side of the page, and the author names on the left side.
This pr implements this, while allowing to reset to the previous layout, by commenting
```yaml
  pagemark:
    left: true
  authors: true 
```
in [index.md](./index.md).

### CODE REVIEW

**Comment by ProphetLamb** [comment]
Warum habe ich das auf Englisch geschrieben? - Gute Frage, nächste Frage

File: index.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ babel-otherlangs:
 
 colorlinks: true # Colorize links, useful for the digital version
 
+pagemark-roman: true # Use roman numerals for all pagemarks
+# index-roman: true # Use roman numerals for indices in content tables
+
+footer:
+  pagemark:  # Determine where the page counter is located, default is center, options are right & left
+    left: true
+  authors: true # Include authors in footer, away from the pagemark.
```
Comment (Siphalor) [inline index.md:56]: I don't think roman pagemarks should be enabled by default.
Also, it should probably be put into the general `footer.pagemark` section.

```suggestion
# index-roman: true # Use roman numerals for indices in content tables

footer:
  pagemark:  # Determine where the page counter is located, default is center, options are right & left
    left: true
    roman: false
  authors: true # Include authors in footer, away from the pagemark.
```

File: index.md [hunk 2]
```diff
@@ -130,6 +138,11 @@ scoop install pandoc pandoc-crossref
 pip install pandoc-acro pandoc-include --user
 ```
 
+[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) können mit GNOME `librsvg` Eingebettet werden. Windows muss diese Library Nachtbeziehen.
+```pwsh
+scoop install https://gist.githubusercontent.com/LuminousPath/55c9a416ab8530cd5875e29e8197f22c/raw/fa5ef794486acfa3d130028b9626d24c7d1ed5ad/rsvg-convert.json
+```
```
Comment (Siphalor) [inline index.md:144]: SVG support isn't really in scope for this pr.
I'm also not really a fan of installing scoop manifests directly, as there won't be support for auto-updates, afaik.

Some buckets with `librsvg` can be found [here](https://scoop.sh/#/apps?q=rsvg&s=2&d=1&o=false).

Also, there are a bunch of grammar/spelling mistakes:

````suggestion
[SVG](https://developer.mozilla.org/en-US/docs/Web/SVG) können mit GNOME `librsvg` eingebettet werden. In Windows muss diese Library manuell nachbezogen werden.
```pwsh
scoop install https://gist.githubusercontent.com/LuminousPath/55c9a416ab8530cd5875e29e8197f22c/raw/fa5ef794486acfa3d130028b9626d24c7d1ed5ad/rsvg-convert.json
```
````

File: pandoc/template.latex [hunk 3]
```diff
@@ -440,7 +448,34 @@ $if(logo)$
...
+$elseif(footer.pagemark.right)$
+\lofoot*{\pagemark}
+\cofoot*{}
+$if(footer.authors)$
+\rofoot*{$for(author)$$author$ $sep$ $endfor$} %\@author
+$endif$ %(footer.authors)
+$else$
+$if(footer.authors)$
+\cofoot*{$for(author)$$author$ $sep$ $endfor$} %\@author
+$endif$ %(footer.authors)
```
Comment (Siphalor) [inline pandoc/template.latex:467]: No support for centered pagemarks?

**Review by Siphalor — CHANGES_REQUESTED** [review]

**Comment by Siphalor** [comment]
First off, I must be blind, because in the official recommendations for the papers, there seems to be no evidence for either footer requirements or Roman numerals.

The official example PDF for bachelor papers uses centered Arabic numerals in the footer.

The part for SVGs doesn't really fit this PR.

**Pull request closed without merging**